docs/user-guide/cli-usingcli.md

```diff
@@ -31,41 +32,331 @@ zowe --ac
+
+  - **Secure credentials in an automation server or container**
+
+    You can set environment variables for use in scripts that run in your CI/CD pipeline. For example, can define environment variables in Jenkins so that your password is not seen in plaintext in logs. You can also define sensitive information in the Jenkins secure credential store.
+
+#### Formatting environment variables
+
+Transform an option into the proper format for a Zowe CLI environment variable, then define a value to the variable. Transform option names according to the following rules:
+
+  - Prefix environment variables with `ZOWE_OPT_`.
+  - Convert lowercase letters in arguments/options to uppercase letters.
+  - Convert hyphens in arguments/options to underscores.
+
+**Tip:** Refer to your operating system documentation for information about how to set and get environment variables. The procedure varies between Windows, Mac, and various versions of Linux.
+
+**Examples:**
+
+The following table provides examples of CLI options and the corresponding environment variable to which you can define a value:
+
+| Command Option          | Environment Variable           | Use Case   |
+| ----------------------- | ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--user`                | `ZOWE_OPT_USER`                | Define your mainframe username to an environment variable to avoid specifying it on all commands or profiles.                           |
+| `--reject-unauthorized` | `ZOWE_OPT_REJECT_UNAUTHORIZED` | Define a value of `true` to the `--reject-unathorized` flag when you always require the flag and do not want to specify it on all commands or profiles. |
+
+#### Setting environment variables in an automation server
+
+You can use environment variables in an automation server, such as Jenkins, to write more efficient scripts and make use of secure credential storage. Automation tools such as Jenkins automation server usually provide a mechanism for securely storing configuration (for example, credentials). In Jenkins, you can use withCredentials to expose credentials as an environment variable (ENV) or Groovy variable.
+
+You can either set environment variables using the `SET` command within your scripts, or navigate to **Manage Jenkins \> Configure System \> Global Properties** and define an environment variable in the Jenkins GUI. For example:
+
+![jenkins gui](../images/guides/CLI/envVarsJenkins.png)
+
+## Using the prompt feature
 
 Zowe CLI lets you enable a command-line "prompt" feature, which masks values on the screen as you type. You might choose to enable the prompt for sensitive credentials such as mainframe username or password.
 
 You can enable the prompt on-demand, or choose to always prompt for a particular option.
 
-### Enable prompt
+#### Enable prompt
```

Comment:
You might want to make some statement that the new automatic prompting feature may provide similar (and easier) functionality for the 4 options that will automatically prompt.